src/cmd/changelog.rs

```diff
@@ -35,6 +36,7 @@ struct ChangeLogTransformer<'a> {
     git: &'a GitHelper,
     context_builder: ContextBuilder<'a>,
     commit_parser: CommitParser,
+    files: &'a Vec<PathBuf>,
```

Comment:
as the changelogtransformer does not need to change the vec, this could use a slice instead:

```suggestion
    files: &'a [PathBuf],
```